pkg/controller/controller_ref_manager.go

```diff
@@ -109,7 +109,7 @@ func GetControllerOf(controllee *v1.ObjectMeta) *metav1.OwnerReference {
 func (m *PodControllerRefManager) AdoptPod(pod *v1.Pod) error {
 	// we should not adopt any pods if the controller is about to be deleted
 	if m.controllerObject.DeletionTimestamp != nil {
-		return fmt.Errorf("cancel the adopt attempt for pod %s because the controlller is being deleted",
+		return fmt.Errorf("Cancel the adopt attempt for pod %s because the controller is being deleted",
```

Comment:
by convention, we tend to have all errors start with lower case, so "cancel" not "Cancel"